src/modules/analytics/analytics.resolver.ts

```diff
@@ -218,4 +221,12 @@ export class AnalyticsResolver {
 
         throw new Error('Invalid parameters');
     }
+
+    @Query(() => GlobalRewardsModel)
+    async globalRewards(
+        @Args() args: GlobalRewardsArgs,
+    ): Promise<GlobalRewardsModel> {
+        const model = await this.globalRewardsService.getGlobalRewards(args.weekOffset);
```

Comment:
I don't think you need a method for this. Just use
`return Object.assign(new GlobalRewardsModel({}), { weekOffset: args.weekOffset });`